### ISSUE TITLE
README: For users having problems with Install on 10.15 or later

### ISSUE THREAD

**Issue by tzugen** [jwise/HoRNDIS#165]
Starting with macOS 10.15 Catalina, Apple introduced a new security feature called System Integrity Protection (SIP) that restricts the loading of certain kexts unless they are properly signed and notarized by Apple. This means that some third-party kexts may not work unless they are updated to meet these new requirements.

You can disable SIP (at your own risk!!)

To disable SIP, boot into Recovery mode by restarting your Mac and holding down the Command + R keys until the Apple logo appears. Once in Recovery mode, open Terminal from the Utilities menu and type the command "csrutil disable" and hit enter. Then restart your Mac normally.

Afterwards you should be able to copy the HoRNDIS.kext to /Library/Extensions

**Comment by ewumnrr** [comment]
Yes, still works on MacBook Pro (M1) with Monterey 12.6 after disabling SIP. Had to be compiled for ARM. Using a Moxee Cellular Hotspot via USB (No Wifi involved).
